frontend/src/components/ManageAccess.vue

```diff
@@ -1,18 +1,146 @@
 <script setup lang="ts">
+import { ref } from 'vue'
+import { ApiService } from '@/services/ApiService';
 import PageTitle from '@/components/PageTitle.vue';
 import router from '@/router';
-import { isApplicationSelected, selectedApplication } from '@/services/ApplicationState';
 import { useToast } from 'vue-toastification';
 
-var foo = 2;
+const apiService = new ApiService()
+
+interface UserRoleAssignment {
+  user_role_xref_id: number,
+  user: {
+    user_type_code: string
+    user_type_desc?:string
+    cognito_user_id: string
+    user_name: string
+  },
+  role: {
+    role_name: string
+    role_type_code: string
+    client_number?: {
```

Comment:
**client_number** is an object seems to be odd.
I am assuming it will be changed from Kevin's return object.

Reply:
Yes, I provided that as feedback as well :)

Reply:
I did suggest Kevin yesterday that if you don't mind, while he is working on real implementation, he can code some "canned data" to return for you right now so you can integrate with the right structure of data from backend and leave the frontend behind when you are done. He will eventually hook up his query and transformation to the json structure you need.
Not sure if that's a good idea you would like, but need to discuss with Kevin.